Makefile

```diff
@@ -19,10 +19,15 @@ OutputName = explorer.exe
 # Commands
 #
 
+all:
+	@make -s build
+	@make -s run
+
 clear:
 	powershell clear
 
 clean:
+	powershell mkdir $(DirOut) -Force
```

Comment:
@OmarBazaraa Do you think this is good? Or should I create a new section for this line (e.g. `setup`)?

Reply:
Does this remove the old data in the `out` directory?

Reply:
No, this is why I kept the 2nd line.